public/index.js

```diff
@@ -1,27 +1,51 @@
+
 /* eslint-disable */
 const main = document.querySelector('main');
-const buttonTryMe = document.getElementById('button__tryme');
+const tryMeButton = document.getElementById('button__tryMe');
 const randomGiftButton = document.getElementById('button__randomGift'); 
 
 let content = localStorage.getItem('content')
   ? JSON.parse(localStorage.getItem('content'))
   : {};
 
+
 /**
  * Check if there is a service worker and register it
```

Comment:
This comment should be moved above the function it refers to (`if('serviceWorker' in navigator)`) .